OpenUpSA/wazimap-ng-ui
# Enable admin to hide indicators of percentage type

## Description

Enable admins to set chart configuration which disables the percentage type. This is usual for cases where values might actually be percentages themselves and we don't want to then convert those to the percentage type

## Related Issue
https://trello.com/c/vnOET23m/815-tooltip-and-table-dont-respect-configuration-that-would-previously-work-to-hide

## How to test it locally
In the admin chart config set for an indicator
```
"defaultType": "Value", 
"disableToggle": true
```
Then check that indicator on the front end

## Screenshots


## Changelog

### Added

### Updated
Admins can now hide percentage types again

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
